lib/viewmodels/home_view_model.dart

```diff
@@ -9,6 +10,7 @@ import "package:proxima/services/geolocation_service.dart";
 /// It fetches the posts from the database and returns a list of
 /// (postId: [PostIdFirestore], postOverview: [PostOverview]) objects to be displayed.
 /// These represent the overview data to be displayed associated to the corresponding post id.
+/// Note: this is also the viewmodel for our post page
```

Comment:
Oh that's true, I didn't see it this way.
I referred to it as the viewmodel of our post page because this is where the data populating the page is "coming from" and I was confused at first when looking for it (I was expecting one viewmodel per page) so I thought it would clarify.
I'll reword it.

Reply:
Reworded in 6dbd8ed386de717bd762f04f3218fd41d8135f23.
Here is what I stated instead: "/// Note: this viewmodel also provides the data for the post page".